language/bytecode_verifier/src/abstract_state.rs

```diff
@@ -73,13 +73,17 @@ enum BorrowInfo {
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct AbstractState {
     locals: BTreeMap<LocalIndex, AbstractValue>,
+    globals: BTreeMap<StructDefinitionIndex, BTreeSet<Nonce>>,
```

Comment:
Do we want an inverse mapping to? Or do we not care about efficiency in that direction?

Reply:
I did not need to lookup in the reverse direction.